dist/Leaflet-WFST.src.js

```diff
@@ -165,11 +165,11 @@ L.Filter.GmlObjectID = L.Filter.extend({
 });
 
 L.Filter.BBox = L.Filter.extend({
-  append: function(bbox, geometryField) {
+  append: function(bbox, geometryField, crs) {
     var filterBBox = L.XmlUtil.createElementNS('ogc:BBOX');
     filterBBox.appendChild(L.XmlUtil.createElementNS('ogc:PropertyName', {}, {value: geometryField}));
 
-    var envelope = L.XmlUtil.createElementNS('gml:Envelope', {srsName: "http://www.opengis.net/gml/srs/epsg.xml#4326"});
+    var envelope = L.XmlUtil.createElementNS('gml:Envelope', {srsName: crs.code});
```

Comment:
Will not work for custom CRS, Need to reproject bbox coordinates wia CRS and not place srsName here.